SourceCode/raytracer/main-win-console/settings.json

```diff
@@ -0,0 +1,28 @@
+{
+    "sceneLibraryDir": "sceneLibrary",
+    "projectDir": "hw13",
+    "targetScenes": ["scene0.crtscene"],
+    "outputDir": "out",
+    
+    "maxDepth": 16,
+    "bias": 0.001,
+    "numBuckets": 12,
+    
+    "debugPixel": false,
+    "debugPixelX": 0,
+    "debugPixelY": 0,
+    
+    "debugLight": false,
+    "debugImageQueue": false,
+    "debugAccelStructure": false,
+    
+    "overrideResolution": false,
+    "resolutionX": 300,
+    "resolutionY": 200,
+    
+    "bWritePng": true,
+    "bWriteBmp": true,
+    
+    "forceNoAccelStructure": false,
```

Comment:
Words like No, Not, None, ... in boolean identifiers tend to confuse the readers often.

Reply:
I completely agree. I'll remove it since I can use `maxAccelTreeDepth` to control this anyway